CiarnuroRT/webapp/index.html

```diff
@@ -39,6 +39,13 @@
                         <input type="submit" tabIndex="4" style="display:none" />
                     </form>
                 </div>
+                <div style="position:absolute;bottom:1rem;right:2rem;line-height:2rem">
```

Comment:
Don't do this, it can mess up the layout if the screen is small or the font size is large. Just put it in the form with the rest of the settings.